tests/integration/data_sources_and_expectations/expectations/test_expect_column_values_to_be_of_type.py

```diff
@@ -66,6 +65,18 @@ def test_success_for_type__INTEGER(batch_for_datasource: Batch) -> None:
     assert result.success
 
 
+@parameterize_batch_for_data_sources(
+    data_source_configs=[
+        BigQueryDatasourceTestConfig(),
+    ],
+    data=DATA,
+)
+def test_success_for_type__INT64(batch_for_datasource: Batch) -> None:
+    expectation = gxe.ExpectColumnValuesToBeOfType(column=INTEGER_COLUMN, type_="INT64")
```

Comment:
🚀

Reply:
Is this a bugfix to the bigquery bigint bug I've heard about?